File: Backend/proyecto/src/main/java/com/admindb/proyecto/modelo/comentarios/ComentarioColumna.java
package com.admindb.proyecto.modelo.comentarios;


import javax.persistence.Column;
import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;

import lombok.Getter;
import lombok.Setter;



@Getter
@Setter
@Entity
@Table(name = "dba_col_comments")
public class ComentarioColumna {
    
    @EmbeddedId
    private ComentarioColumnaId id;

    @Column(name = "comments")
    private String comentario;
}

File: Frontend/frontend-adminbd/src/app/common/comentario-columna.ts
import { ComentarioColumnaId } from "./comentario-columna-id";

export class ComentarioColumna {
    id: ComentarioColumnaId;

    comentario: string;

    constructor(id: ComentarioColumnaId, comentario: string){
        this.id = id;
        this.comentario = comentario;
    }
}
